tests/fixtures/vault.py

```diff
@@ -52,7 +52,7 @@ def vault_fixture() -> Generator[VaultFixture, None, None]:
         host = vault_container.get_container_host_ip()
         port = vault_container.get_exposed_port(VAULT_PORT)
         role_id, secret_id = configure_vault(host=host, port=int(port))
-        config = VaultConfig(
+        config = VaultConfig(  # type: ignore
```

Comment:
Not sure about that, mypy complains that `vault_kube_role` and `service_account_token_path` are not passed, howver they have both a default.

Reply:
Thanks! I will update this.